cmd/svcat/broker/sync_cmd.go

```diff
@@ -20,24 +20,26 @@ import (
 	"fmt"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 	"github.com/spf13/cobra"
 )
 
 type syncCmd struct {
-	*command.Context
+	*command.Namespaced
```

Comment:
I believe you should also add `*command.Scoped` here. 

On a related note: I find it weird that `Scoped` doesn't include `Namespaced`, since every Scoped command is also Namespaced. @carolynvs ?

Reply:
I can change it and I might need dig the question as well, The Scoped and Namespaced is a little bit confusing to me as well...

Reply:
Yes, let's do that in a follow-up PR, when @carolynvs confirms we can do that.

Reply:
Scoped and Namespaced evolved separately as more commands supported things. Namespaced and scoped don't always intersect, e.g. for service instances they are namespaced but never scoped. That's why they weren't collapsed into a single interface.